### PR TITLE
gNMI-1.10 updates

### PR DESCRIPTION
**Pull request by LimeHat**
- Update TestHardwarePort function to accomodate vendor-specific component naming

---
<sup>
This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia’s intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind.</sup>

### CODE REVIEW

File: feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go [hunk 1]
```diff
@@ -437,7 +444,7 @@ func TestComponentParent(t *testing.T) {
 func TestSoftwareVersion(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 
-	cards := components.FindComponentsByType(t, dut, cpuType)
+	cards := components.FindComponentsByType(t, dut, supervisorType)
```
Comment (xw-g) [inline feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:447]: @mojiiba I think this changes looks good to me. Any objection? Thanks. 

cc: @vosipchu

**Comment by xw-g** [comment]
Per offline chat, the `cpuType` update will be a separate PR for discussion.

**Comment by LimeHat** [comment]
Updated per the comment above

**Comment by LimeHat** [comment]
The other change moved into #1279

**Review by xw-g — APPROVED** [review]

**Comment by xw-g** [comment]
All changes are about one vendor. Merging this PR upon my own LGTM.

**Pull request merged**